App crashing on Creating new events

**Is it a ?**

- [x] Bug Report

- [ ] Feature Request
    
- [ ] Chore
    
### Bug Report    

**Actual Behaviour**
When a new event is being created, the app is crashing.

**Steps to reproduce it**
Go to Events and select the FAB to create new events

**LogCat for the issue**
``
 Caused by: java.lang.IllegalArgumentException: No injector was found for org.fossasia.openevent.app.core.event.create.CreateEventFragment
        at dagger.android.support.AndroidSupportInjection.findHasFragmentInjector(AndroidSupportInjection.java:92)
        at dagger.android.support.AndroidSupportInjection.inject(AndroidSupportInjection.java:57)
        at org.fossasia.openevent.app.common.di.AppInjector$2.onFragmentCreated(AppInjector.java:78)
        at android.support.v4.app.FragmentManagerImpl.dispatchOnFragmentCreated(FragmentManager.java:3474)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1380)
        at android.support.v4.app.FragmentTransition.addToFirstInLastOut(FragmentTransition.java:1188)
        at android.support.v4.app.FragmentTransition.calculateFragments(FragmentTransition.java:1071)
        at android.support.v4.app.FragmentTransition.startTransitions(FragmentTransition.java:115)
        at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2379)
        at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2337)
        at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2244)
        at android.support.v4.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3255)
        at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:3205)
        at android.support.v4.app.FragmentController.dispatchActivityCreated(FragmentController.java:195)
        at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:597)
        at android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:177)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1238)
        at android.app.Activity.performStart(Activity.java:6374)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2418)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
        at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:157) 
        at android.app.ActivityThread.main(ActivityThread.java:5571) 
        at java.lang.reflect.Method.invoke(Native Method)
``

**Would you like to work on the issue?**
Yes!

--- a/app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventActivity.java
+++ b/app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventActivity.java
@@ -2,12 +2,22 @@ package org.fossasia.openevent.app.core.event.create;
 
 import android.os.Bundle;
 import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
 import android.support.v7.app.AppCompatActivity;
 import android.view.MenuItem;
 
 import org.fossasia.openevent.app.R;
 
-public class CreateEventActivity extends AppCompatActivity {
+import javax.inject.Inject;
+
+import dagger.android.AndroidInjector;
+import dagger.android.DispatchingAndroidInjector;
+import dagger.android.support.HasSupportFragmentInjector;
+
+public class CreateEventActivity extends AppCompatActivity implements HasSupportFragmentInjector {
+
+    @Inject
+    DispatchingAndroidInjector<Fragment> fragmentDispatchingInjector;
 
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
@@ -30,5 +40,10 @@ public class CreateEventActivity extends AppCompatActivity {
             default:
         }
         return super.onOptionsItemSelected(item);
+    }
+
+    @Override
+    public AndroidInjector<Fragment> supportFragmentInjector() {
+        return fragmentDispatchingInjector;
     }
 }

